src/test/scala/mesosphere/marathon/test/Mockito.scala

```diff
@@ -20,6 +20,7 @@ trait Mockito extends MockitoSugar {
   def times(num: Int) = M.times(num)
   def timeout(millis: Int) = M.timeout(millis.toLong)
   def atLeastOnce = M.atLeastOnce()
+  def once = M.times(1)
```

Comment:
`times(1)` is the default, so this is redundant, but ok if you want to be very explicit =)